test/dust-test.js

```diff
@@ -18,10 +19,13 @@ describe("bundalo dust bundler @dust@", function () {
 			if (err) {
 				return done(err);
 			}
-			if (data.greeting && bundloo.__cache()[path.normalize('nest/dusta.properties')] && !bundlee.__cache()[path.normalize('nest/dusta.properties')]) {
+			try {
+				assert.ok(data.greeting);
+				assert.ok(bundloo.__cache()[path.normalize('nest/dusta.properties')]);
+				assert.ok(!bundlee.__cache()[path.normalize('nest/dusta.properties')]);
```

Comment:
Except this isn't in the test callback, it's asynchronous from it. Which means if it's caught, it's via domains, and ew.